src/functions/schema.ts

```diff
@@ -65,7 +80,8 @@ function getSchemaId(schemaObj: JSONSchema): void | string {
 }
 
 const validators = new (class extends WeakMap<JSONSchema, ValidateFunction> {
-  public get(schemaObj: JSONSchema) {
+  public get(schemaObj: JSONSchema, oasVersion: number = 0) {
```

Comment:
```suggestion
  public get(schemaObj: JSONSchema, oasVersion: Optional<number>) {
```
Let's just make it optional. 0 looks a tad weird.